src/signTxOutput.c

```diff
@@ -404,6 +432,7 @@ static void signTxOutput_handleAssetGroupAPDU(uint8_t* wireDataBuffer, size_t wi
 		);
 		TRACE();
 	}
+	subctx->stateData.token.assetNameSize = ASSET_NAME_SIZE_MAX + 1;	//maybe turn it into a signed number and use -1? takes up too much space anyways
```

Comment:
I am sure we do not want to use raw int for buffer size in just a single place.

I don't understand this assignment: why is it in this place? And I really dislike using buffer size that exceeds the max. Can we use `subctx->currentToken` to distinguish whether we are at the first policyId or not?